definitions/private-law/json/CaseEventToFields.json

```diff
@@ -1,23 +1,22 @@
 [
   {
-    "LiveFrom": "01/07/2021",
+    "LiveFrom": "05/11/2021",
     "CaseTypeID": "C100",
     "CaseEventID": "solicitorCreate",
-    "CaseFieldID": "ApplicantName",
+    "CaseFieldID": "ApplicantCaseName",
     "DisplayContext": "MANDATORY",
     "PageID": 1,
     "PageDisplayOrder": 1,
     "ShowSummaryChangeOption": "Yes"
   },
   {
-    "LiveFrom": "01/07/2021",
+    "LiveFrom": "08/11/2021",
     "CaseTypeID": "C100",
-    "CaseEventID": "solicitorCreate",
-    "CaseFieldID": "ChildName",
+    "CaseEventID": "CaseName",
+    "CaseFieldID": "ApplicantCaseName",
     "DisplayContext": "MANDATORY",
     "PageID": 1,
-    "PageDisplayOrder": 1,
-    "ShowSummaryChangeOption": "Yes"
+    "PageDisplayOrder": 2
```

Comment:
If there is only one field on the event screen shouldn't this be 1?

Reply:
Display order changed to 1.